src/style.css

```
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+  font-style: normal;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  margin: 20px 8px;
+}
+
+.page-search {
+  position: relative;
+}
+
+.input--big {
+  box-sizing: border-box;
+  width: 100%;
```

```suggestion
  width: 100%;
```

both inputs have same styles like this one
make a class that contains all shared styles and use it for both inputs